spec/bundler/dsl_spec.rb

```diff
@@ -271,7 +271,7 @@
     it "will raise a Bundler::GemfileError" do
       gemfile "gem 'foo', :path => /unquoted/string/syntax/error"
       expect { Bundler::Dsl.evaluate(bundled_app("Gemfile"), nil, true) }.
-        to raise_error(Bundler::GemfileError, /There was an error parsing `Gemfile`:( compile error -)? unknown regexp options - trg. Bundler cannot continue./)
+        to raise_error(Bundler::GemfileError, /There was an error parsing `Gemfile`:( compile error -)? unknown regexp options - trg.+? Bundler cannot continue./)
```

Comment:
Isn't the `?` character unnecessary? `+` already expresses optionality.

Reply:
Can't keep regex engine differences straight, so I often use lazy (non-greedy) ...  Regardless, removed, passed.